src/plugins/overlay/reports/a11y-1-fr.hbs

```diff
@@ -0,0 +1,77 @@
+---
+{
+	"title": "Assessment d'accessibilité #3 - Contenu superposé",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
+	"tag": "overlay",
+	"parentdir": "overlay",
+	"parentPage": "Contenu superposé",
+	"parentPageURL": "overlay",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2022-08-19"
+}
+---
+<div id="ds-a11y" data-wb-jsonmanager='{ "url": "a11y-1.json", "name": "a11y" }'></div>
+
+<dl class="dl-horizontal">
+	<dt>Identifiant du rapport d'évaluation</dt>
+	<dd data-json-replace="#[a11y]/id" lang="en"><span lang="fr">Non disponible</span></dd>
+	<dt>Type</dt>
+	<dd>Évaluation de l'accessibilité</dd>
+	<dt>Évalué par:</dt>
+	<dd data-json-replace="#[a11y]/assertedBy" lang="en"><span lang="fr">Non disponible</span></dd>
+	<dt>Date:</dt>
+	<dd data-json-replace="#[a11y]/date" lang="en"><span lang="fr">Non disponible</span></dd>
+	<dt>Page:</dt>
+	<dd><a data-wb-json='[
+
+			{
+				"url": "#[a11y]/subjectUrl",
+				"type": "attr",
+				"attr": "href"
+			},
+			{
+				"url": "#[a11y]/subjectUrl",
+				"type": "replace"
+			}
+
+		]' href="#" lang="en"><span lang="fr">Non Définie</span></a></dd>
+	<dt>Subject</dt>
```

Comment:
- Should we used the French term: "Sujet" instead?
- In terms of the output, it's stated: "Evaluated all Success Criteria at Level AA of WCAG 2.2"...is this statement should mentioned that it's evaluated against criteria Level AA of WCAG 2.0, 2.1 and 2.2?
- IS there any French translation for the output message?